packages/lib/shared/utils/error-filters.ts

```diff
@@ -56,3 +63,20 @@ export function isUnbalancedAddErrorMessage(error: Error | null): boolean {
 
   return errorStrings.some(hasErrors)
 }
+
+export function isInvariantRatioSimulationErrorMessage(errorMessage?: string): boolean {
+  if (errorMessage?.includes('InvariantRatioAboveMax')) return true
+  if (errorMessage?.includes('InvariantRatioBelowMin')) return true
```

Comment:
use or?

Reply:
I don't have an strong opinion unless the options would grow.
Would you prefer `or` or this shorter version: 
`['InvariantRatioAboveMax', 'InvariantRatioBelowMin'].includes(errorMessage)` ?